File: kartwheel/app/src/main/java/us/handstand/kartwheel/fragment/AlreadyClaimedFragment.java
package us.handstand.kartwheel.fragment;


import android.os.Bundle;
import android.support.annotation.IdRes;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import us.handstand.kartwheel.R;
import us.handstand.kartwheel.activity.TicketActivity;

public class AlreadyClaimedFragment extends Fragment implements TicketActivity.TicketFragment, View.OnClickListener {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        ViewGroup fragmentViewGroup = (ViewGroup) inflater.inflate(R.layout.fragment_already_claimed, container, false);
        fragmentViewGroup.findViewById(R.id.already_claimed_link).setOnClickListener(this);
        return fragmentViewGroup;
    }

    @Nullable
    @Override
    public Bundle onButtonClicked(@IdRes int id) {
        return null;
    }

    @Override
    public void onClick(View v) {
        // TODO: Where do we send them?
//        startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("")));
    }
}

File: kartwheel/app/src/main/kotlin/us/handstand/kartwheel/fragment/AlreadyClaimedFragment.kt
package us.handstand.kartwheel.fragment


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import us.handstand.kartwheel.R
import us.handstand.kartwheel.activity.TicketActivity

class AlreadyClaimedFragment : Fragment(), TicketActivity.TicketFragment, View.OnClickListener {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val fragmentViewGroup = inflater!!.inflate(R.layout.fragment_already_claimed, container, false) as ViewGroup
        fragmentViewGroup.findViewById(R.id.already_claimed_link).setOnClickListener(this)
        return fragmentViewGroup
    }

    override fun onClick(v: View) {
        if (v.id == R.id.already_claimed_link) {
            // TODO: Where do we send the user?
            //        startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("")));
        }
    }
}
